bin/pytorch_inference/evaluate.py

```diff
@@ -1,181 +1,116 @@
+'''
+Reads a TorchScript model from file and restores it to the C++
+app, together with the encoded tokens from the input_tokens
+file.  Then it checks the model's response matches the expected.
+
+This script first prepares the input files, then launches the C++
+pytorch_inference program which handles them in batch.
+
+Run this script with input from one of the example directories,
+for example:
+
+python3 evaluate.py /path/to/conll03_traced_ner.pt examples/ner/input.json  examples/ner/expected_response.json
+'''
+
 import argparse
 import json
 import os
 import platform
 import stat
 import subprocess
-import time
 
-
-'''
-Reads a TorchScript model from file and streams it via a pipe
-to the C++ app. Once the model is loaded the script sends the 
-encoded tokens from the input_tokens files and checks the model's 
-response matches the expected. 
-
-This script first lauches the C++ pytorch_inference process then
-connects the pipes.
-
-Then run this script with input from one of the example directories
-
-python3 evaluate.py /path/to/conll03_traced_ner.pt examples/ner/input.json  examples/ner/expected_response.json
-
-'''
 def parse_arguments():
     parser = argparse.ArgumentParser()
     parser.add_argument('model', help='A TorchScript model with .pt extension')
     parser.add_argument('input_tokens', help='JSON file with an array field "tokens"')
     parser.add_argument('expected_output', help='Expected output. Another JSON file with an array field "tokens"')
-    # The app pipe names
-    parser.add_argument('--restore_pipe', default='restore_pipe')
-    parser.add_argument('--input_pipe', default='input_pipe')
-    parser.add_argument('--output_pipe', default='output_pipe')
-    parser.add_argument('--log_pipe', default='log_pipe')
+    parser.add_argument('--restore_file', default='restore_file')
+    parser.add_argument('--input_file', default='input_file')
+    parser.add_argument('--output_file', default='output_file')
 
     return parser.parse_args()
 
-
 def path_to_app():
 
-	os_platform = platform.system()
```

Comment:
I changed tabs to spaces in this file as it had a mix before - best to review with whitespace differences hidden